lib_ansi/escape_parser.ml

```diff
@@ -76,6 +92,7 @@ let read_intermediates ~params start =
     | Some (x, s2) when is_final_byte x -> (
         let func = Stream.(start -- s2 |> string_of_span) in
         let params = Astring.String.cuts ~sep:";" params in
+        let params = List.concat_map (Astring.String.cuts ~sep:":") params in
```

Comment:
This use of List.concat_map pulls the supported OCaml version up by a couple. I think we're ok with OCaml 4.10+ for ocurrent, but @talex5 will confirm.